src/Private/Invoke-PSTypeFormatter.ps1

```diff
@@ -0,0 +1,78 @@
+function Invoke-PSTypeFormatter {
+    <#
+    .SYNOPSIS
+        Helps formatting function files to dbatools' standards
+
+    .DESCRIPTION
+        Uses PSSA's Invoke-Formatter to format the target files and saves it without the BOM.
+
+    .PARAMETER Path
+        The path to the ps1 file that needs to be formatted
+
+    .NOTES
+        Author: Simone Bizzotto
+
+    .EXAMPLE
+        PS C:\> Invoke-PSTypeFormatter -Path C:\dbatools\functions\Get-DbaDatabase.ps1
+
+        Reformats C:\dbatools\functions\Get-DbaDatabase.ps1 to dbatools' standards
+
+    #>
+    [CmdletBinding()]
+    param (
+        [parameter(Mandatory, ValueFromPipeline)]
+        [object[]]$Path
```

Comment:
This could be a FileInfo type and it'll handle both pipeline input from gci etc and strings.  It might also be worth adding an alias for PSPath if we were to add ValueFromPipelineByProperty.

Reply:
😮  fileinfo handles both?! awesome! i did not know

Reply:
I got it working in an example and with a copy of this function by doing `[System.IO.FileInfo[]]$Path` which will happily allow me to pass in any string that looks like a path and it'll work with it correctly.  I did only test that on v5 but I'd expect it to work on 3 and 4.